pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -254,14 +266,50 @@ func fitsRequest(podRequest *preFilterState, nodeInfo *schedulernodeinfo.NodeInf
 	return insufficientResources
 }
 
+func isIgnoredResource(name string, ignoredResources, ignoredGroups sets.String) bool {
+	if ignoredResources.Has(name) {
+		return true
+	}
+
+	for group := range ignoredGroups {
+		// We know that name is the extended resource, so it is divided by "/".
+		// group matches the substring before "/".
+		if regexp.MustCompile("^" + group).MatchString(name) {
+			return true
+		}
+	}
+	return false
+}
+
+func validateFitArgs(args *FitArgs) error {
+	var allErrs field.ErrorList
```

Comment:
Can you also validate the existing `ignoredResources`?

Reply:
done